sentences/fr-CA/fan_HassTurnOff.yaml

```diff
@@ -3,11 +3,11 @@ intents:
   HassTurnOff:
     data:
       - sentences:
-          - <eteins> ([tous] | [toutes] ) <ventilateurs> <dans> [<area>]
+          - "<eteins> ([tous]|[toutes]) <ventilateurs> <dans> [<area>]"
```

Comment:
If `toutes and tous` are optional why not change it to just
` "<eteins> [(tous|toutes)] <ventilateurs> <dans> [<area>]"